managed/services/inventory/services.go

```diff
@@ -76,6 +76,40 @@ func (ss *ServicesService) List(ctx context.Context, filters models.ServiceFilte
 	return res, nil
 }
 
+// ListActiveServiceTypes lists all active Service Types
+//
+//nolint:unparam
+func (ss *ServicesService) ListActiveServiceTypes(ctx context.Context) ([]inventorypb.ServiceType, error) {
+	var types []models.ServiceType
+	err := ss.db.InTransaction(func(tx *reform.TX) error {
+		var err error
+		types, err = models.FindActiveServiceTypes(tx.Querier)
+		return err
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	res := make([]inventorypb.ServiceType, 0, len(types))
+	for _, t := range types {
+		switch t {
+		case models.MySQLServiceType:
+			res = append(res, inventorypb.ServiceType_MYSQL_SERVICE) //nolint:nosnakecase
+		case models.MongoDBServiceType:
+			res = append(res, inventorypb.ServiceType_MONGODB_SERVICE) //nolint:nosnakecase
+		case models.PostgreSQLServiceType:
+			res = append(res, inventorypb.ServiceType_POSTGRESQL_SERVICE) //nolint:nosnakecase
+		case models.ProxySQLServiceType:
+			res = append(res, inventorypb.ServiceType_PROXYSQL_SERVICE) //nolint:nosnakecase
+		case models.HAProxyServiceType:
+			res = append(res, inventorypb.ServiceType_HAPROXY_SERVICE) //nolint:nosnakecase
+		case models.ExternalServiceType:
+			res = append(res, inventorypb.ServiceType_EXTERNAL_SERVICE) //nolint:nosnakecase
+		}
```

Comment:
It seems that we can just remove this switch-case and just add every `t` to result

Reply:
`t` is of type `string` and `res` requires `int`. It appears to me there needs to be a way to map the values to each other.
How would you suggest we use just `t`? Maybe I'm missing something.

Reply:
Ah, yes, my bad. Nevermind.